src/modules/keyboardmanager/ui/LoadingAndSavingRemappingHelper.cpp

```diff
@@ -19,7 +19,7 @@ namespace LoadingAndSavingRemappingHelper
             std::wstring appName = remappings[i].second;
 
             bool ogKeyValidity = (ogKey.index() == 0 && std::get<DWORD>(ogKey) != NULL) || (ogKey.index() == 1 && std::get<Shortcut>(ogKey).IsValidShortcut());
-            bool newKeyValidity = (newKey.index() == 0 && std::get<DWORD>(newKey) != NULL) || (newKey.index() == 1 && std::get<Shortcut>(newKey).IsValidShortcut());
+            bool newKeyValidity = (newKey.index() == 0) || (newKey.index() == 1 && std::get<Shortcut>(newKey).IsValidShortcut());
```

Comment:
We should ideally change the key code of None to some other value so that this change can be reverted. I think the max value of DWORD makes sense since we definitely wont use that for anything in the future. We just have to define VK_NONE in the constants

Reply:
I agree with this. The above code in PowerToys/src/common/keyboard_layout.cpp is also tied to None not being a proper code/being treated differently than the other codes in the list.